sa-server/index.js

```diff
@@ -0,0 +1,95 @@
+import express from 'express';
+import bodyParser from 'body-parser';
+import dotenv from 'dotenv';
+const app = express();
+import mongoose from "mongoose";
+import swaggerUi   from "swagger-ui-express";
+import swaggerSpec from "./swagger.js";
+import cookieParser from'cookie-parser';
+app.use(cookieParser());
+app.use(express.static("public"));
+dotenv.config();
+app.set("view engine", "ejs");
+app.use("/api-docs", swaggerUi.serve, swaggerUi.setup(swaggerSpec));
+app.use(bodyParser.urlencoded({extended: true}));
+mongoose.connect(process.env.api)

```

Comment:
Also separate database configuration. Create a new directory config/dbconfig.js and then write the database configuration codes in it.